_config.yml

```diff
@@ -1 +1 @@
-exclude: [src]
+exclude: ["src","highlight","hooks"]
```

Comment:
"highlight" should not be in there, because this version of the docs still uses the highlight.js file